### PR TITLE
Added command starting the website to README

### LINKED ISSUE #100: Start website locally
Is there an easy way to start the website locally? There doesn't seem to be a script that does it.

It's also quite difficult to just start a local server because all paths in the website itself are prefixed with "/eslint-plugin-regexp/". Removing that prefix will cause Vue to generate different _absolute_ paths. This enables me that start the website locally but it will break the live website.

### PR DESCRIPTION
**Pull request by RunDevelopment**
This resolves #100.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ module.exports = {
 
 This plugin provides one config:
 
-- `plugin:regexp/recommended` ... This is the recommended configuration for this plugin.
```
Comment (RunDevelopment) [inline README.md:70]: My editor automatically trims trailing spaces. Should I revert these changes?

Comment (ota-meshi) [reply README.md:70]: If we want to break a line, you need two spaces, but I think we don't have to break a line here 😄

**Review by ota-meshi — APPROVED** [review]
Thank you!

File: README.md (hunk 1 above)
Comment (RunDevelopment) [reply README.md:70]: Is that a common Markdown feature or specific to GH-flavor Markdown? 

I always use `<br>` for line breaks btw. It's more explicit.
